pkg/controller/service/handlers.go

```diff
@@ -140,7 +138,65 @@ func (s *Service) register(w http.ResponseWriter, r *http.Request) {
 			}
 		}
 
-		utils.Respond(w, user)
```

Comment:
Can you keep this? The UI does a check to see whether registration confirmation is required.

Reply:
Does it do it on both `s.email==nil` and the else block, or just if `s.email==nil`? We could just create a new session or respond with the user if `s.email==nil`.